x-pack/platform/plugins/shared/alerting/server/application/backfill/methods/schedule/schemas/schedule_backfill_params_schema.ts

```diff
@@ -12,13 +12,23 @@ import { MAX_SCHEDULE_BACKFILL_BULK_SIZE } from '../../../../../../common/consta
 export const scheduleBackfillParamSchema = schema.object(
   {
     ruleId: schema.string(),
-    start: schema.string(),
-    end: schema.maybe(schema.string()),
+    ranges: schema.arrayOf(
+      schema.object({
+        start: schema.string(),
+        end: schema.string(),
+      })
+    ),
     runActions: schema.maybe(schema.boolean()),
   },
   {
-    validate({ start, end }) {
-      return validateBackfillSchedule(start, end);
+    validate({ ranges }) {
+      const errors = ranges
```

Comment:
Yeah, ditto on just calling `validateBackFillRanges`, here, so we don't have this logic duplicated.